src/otx/algo/classification/utils/ignored_labels.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Util functions related to ignored_labels."""
+
+from __future__ import annotations
+
+import torch
+
+from otx.core.data.entity.base import ImageInfo
+
+
+def get_valid_label_mask(img_metas: list[ImageInfo], num_classes: int) -> torch.Tensor:
```

Comment:
Could you add unit test for this class?

Reply:
Thanks! Done - https://github.com/openvinotoolkit/training_extensions/commit/5ed3863c2f7d3db9b94acbbf9530abbce28536ba